lib/child_process.js

```diff
@@ -42,20 +42,28 @@ const {
   SafeSet,
   StringPrototypeSlice,
   StringPrototypeToUpperCase,
+  PromiseReject,
 } = primordials;
 
 const {
   promisify,
   convertToValidSignal,
   getSystemErrorName
 } = require('internal/util');
+
 const { isArrayBufferView } = require('internal/util/types');
 let debug = require('internal/util/debuglog').debuglog(
   'child_process',
   (fn) => {
     debug = fn;
   }
 );
+let DOMException;
+function lazyDOMException(message) {
```

Comment:
We've decided not to reject with lazyDOMException but with AbortError (like the regular `spawn`)

Reply:
Sure, done in 0db89e4 👍